logging/logging.go

```diff
@@ -0,0 +1,43 @@
+// Package logging provides a logger, as well as configuration.
+package logging
+
+import (
+	"github.com/giantswarm/request-context"
+)
+
+var (
+	logger *requestcontext.Logger
+
+	// The default log level to use.
+	logLevel = "INFO"
+	// Whether to use color, by default.
+	useColor = false
+)
+
+// SetLogLevel sets the log level of the logger to be created.
+func SetLogLevel(level string) {
+	logLevel = level
```

Comment:
I did not follow the logger discussions. What were the decisions? Why do we want this here? Why not simply glog? It has nice verbosity features. 

FYI I had this logging problem in one of my own projects as well recently and I wrote my own logger supporting filtering for log levels, verbosity and objects emitting the log messages.